bespin/option_spec/stack_objs.py

```diff
@@ -22,6 +22,12 @@
 import os
 import re
 
+# python 2.6 set compat
+try:
+    set
+except NameError:
+    from sets import Set as set
```

Comment:
Confirmed, think I must be confusing with 2.3-2.4 compat
https://docs.python.org/2.6/library/stdtypes.html#set